install_config/configuring_sdn.adoc

```diff
@@ -211,6 +211,50 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[migrating-between-sdn-plugins-networkpolicy]]
+=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+
+If you want to migrate from *ovs-multitenant* to *ovs-networkpolicy*, you will first need
```

Comment:
From 217 to 222 we can simplify these with:
```
Before you migrate from *ovs-multitenant* to *ovs-networkpolicy*:

* Verify that every namespace has a unique `NetID`. 
* If you have  xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined
 projects together] or  xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made projects global], undo those changes before proceeding.
```

Reply:
"verify that every namespace has a unique NetID" and "undo those changes" are the same thing expressed in two different ways. I tried rewording it some. See if it's better now.